### DISCUSSION TITLE
RLC-510A (IPC_523128M5MP_V2) v3.1.0.2368_23062700

### DISCUSSION THREAD

**Discussion by crycode-de** [AT0myks/reolink-fw-archive#41] [Firmware sharing]
Hi, from the Reolink support I got a link to the latest firmware for the RLC-510A (IPC_523128M5MP_V2).

| | |
|---|---|
| Model | RLC-510A |
| Build No. | build 23062700 |
| Hardware No. | IPC_523128M5MP_V2 |
| Config Version | v3.1.0.0 |
| Firmware Version | v3.1.0.2368_23062700 |
| Web Interface | v1.0.284 |

https://support-d.reolink.com/attachments/token/w6qOgWWCEs7LAa7j582OJfD2u/?name=IPC_523128M5MP_V2.2368_23062700.RLC-510A.OV05A10.5MP.REOLINK.pak

This firmware adds pet detection and also an updated web interface.

After the firmware update I needed to reload the web interface using CRTL+F5 to purge the local browser cache.
(The "Device is upgrading..." dialog stayed at 100% for some time but the update was complete. Reload the page manually to proceed if needed.)

**Comment by AT0myks** [comment]
Thanks, added in 7ef3395634c7587a4626352817e5cc5de8f13bec.

**Reply by tsaoi** [reply]
Hi team, unfortunately the link is no longer available - asks for a Reolink Support login and then errors out as the token has expired. 

I've contacted support to provide and they keep linking me to the non-V2 version.. even after specifying model, HW and firmware filename?!? @crycode-de @AT0myks

**Reply by tsaoi** [reply]
@AT0myks I managed to get hold of the file direct from Reolink support. Attached here FYI (to extract, rename to *.7z)

[IPC_523128M5MP_V2.2368_23062700.RLC-510A.OV05A10.5MP.REOLINK (2).zip](https://github.com/user-attachments/files/19676294/IPC_523128M5MP_V2.2368_23062700.RLC-510A.OV05A10.5MP.REOLINK.2.zip)

**Reply by jihanski** [reply]
@AT0myks @tsaoi @crycode-de their support is not responding so hoping i can confirm here; i am looking for the RLC-520A V2 but download link says access denied- finding the version pointed me to here. Please advice, thank you.

**Reply by CrashOverrideProductions** [reply]
> @AT0myks I managed to get hold of the file direct from Reolink support. Attached here FYI (to extract, rename to *.7z)
> 
> [IPC_523128M5MP_V2.2368_23062700.RLC-510A.OV05A10.5MP.REOLINK (2).zip](https://github.com/user-attachments/files/19676294/IPC_523128M5MP_V2.2368_23062700.RLC-510A.OV05A10.5MP.REOLINK.2.zip)

You are a legend, I was about to go and rip all my 520A-V2s down and throw them into fire

**Reply by SebSeb11** [reply]
@tsaoi thank you! Works with 520A-V2, even if model and camera symbol in app changed to _RLC-510A_.
